cmd/devstream/show.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/devstream-io/devstream/internal/pkg/show"
+	"github.com/devstream-io/devstream/pkg/util/log"
+)
+
+var plugin string
+
+var showCMD = &cobra.Command{
+	Use:   "show",
```

Comment:
Use: "show [plugin]"

This will print:

```shell
Usage:
  dtm show [plugin] [flags]
```

In the future, we could have:

```shell
Usage:
  dtm show [plugin|outputs|xxx] [flags]
```

Reply:
👍